erpnext_com/www/docs/user/manual/en/accounts/promotional-scheme.md

```diff
@@ -0,0 +1,64 @@
+# Promotional Scheme
+
+Promotional schemes are the key to successful retail stores. Promotional schemes can be easily configure in the erpnext. Promotional scheme is linked to the pricing rule, against each slab system will generate the pricing rule.
+
+# 1. How to create a Promotional Scheme
+
+1. Go to: **Selling > Items and Pricing > Promotional Scheme > New**.
+1. Set a title for the rule.
+1. Select what to Apply On like Item Code, Item Group, Brand etc.
+1. Based on the apply on system will give you the provision to select the Item Code / Item Group / Brand in the table.
```

Comment:
Based on the 'Apply On', system will give you the provision to select the Item Code / Item Group / Brand in the table.